src/mongocxx/options/find.hpp

```diff
@@ -270,6 +294,7 @@ class MONGOCXX_API find {
     stdx::optional<std::string> _comment;
     stdx::optional<enum cursor_type> _cursor_type;
     stdx::optional<std::int32_t> _limit;
+    stdx::optional<std::int64_t> _max_await_time_ms;
```

Comment:
Hmm. Good catch! That is because nothing does.

This will only be used to default the maxTimeMS at the cursor level for getMore commands. I will talk to the C driver team and figure out how they plan to implement this on their end :)